### DISCUSSION TITLE
Can this be converted into a Nuxt module ?

### DISCUSSION THREAD

**Discussion by mohdlatif** [scottyzen/woonuxt#151] [Q&A]
This repository is excellent, but I'm concerned about the time it will take to update my code. Let's say I download this repository, customize it for a client, and then, a few months later, I need to update the code due to a bug or because of a new added feature. Since the code in the GitHub repository has been updated, what approach should I take here?

**Comment by scottyzen** [comment]
Hey,

I'm glad you like the repository. I've set it up so that you can customize
any part of the site. Just like you would with a WordPress theme and its
child theme. So you would just override the parts of the site that you want
to customize. This way, you can update the code in the GitHub repository
without losing your customizations. I'm probably going to add the
`woonuxt_base` folder to a repo (example
<https://nuxt.com/docs/guide/going-further/layers#git-repository>) of its
own and have the demo repo separate. But for now, overriding the components
and pages is the way to go.

Kind regards
Scott
